src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmDebuggingExtension/.npmrc

```diff
@@ -0,0 +1,4 @@
+# Auto generated file from Gardener Plugin CentralFeedServiceAdoptionPlugin
+registry=https://pkgs.dev.azure.com/dnceng/internal/_packaging/internal_PublicPackages/npm/registry/
```

Comment:
@mmitche I assume these are the "public" feeds you were [referring to](https://dev.azure.com/dnceng/internal/_git/dotnet-razor-tooling/pullrequest/23705)?

Reply:
Nope, should use `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public-npm/npm/registry/`, and remove always_auth=true. The PR generated is using an internal feed.

then:
1. `npm install -g vsts-npm-auth --registry https://registry.npmjs.com --always-auth false`
2. `vsts-npm-auth -config .npmrc`
3. Clear cache
4. npm install.

Reply:
Ah ok, thanks @mmitche! I've gone ahead and updated after doing all the above.

Will folks on the team need to do a similar flow of steps to build everything in the future?

Reply:
Nope, unless they are ingesting new package versions.